### PR TITLE
docs/admin: update steps of adding repositories

### LINKED ISSUE #8288: External services documentation needs to be updated
No external services link in site admin anymore. 

https://docs.sourcegraph.com/admin/external_service/github#configuration

### PR DESCRIPTION
**Pull request by unknwon**
This PR updates steps for adding repositories to the Sourcegraph instance.

Fixes #8288.

### CODE REVIEW

File: doc/admin/external_service/aws_codecommit.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ Site admins can sync Git repositories hosted on [AWS CodeCommit](https://aws.ama
 To set this up, add AWS CodeCommit as an external service to Sourcegraph:
```
Comment (ryan-blunden) [inline doc/admin/external_service/aws_codecommit.md]:
```suggestion
To connect AWS CodeCommit to Sourcegraph:
```

I would also apply this style to the other code host instructions.

File: doc/admin/external_service/aws_codecommit.md [hunk 2]
```diff
@@ -5,8 +5,8 @@ Site admins can sync Git repositories hosted on [AWS CodeCommit](https://aws.ama
...
 
 1. Go to **User menu > Site admin**.
```
Comment (ryan-blunden) [inline doc/admin/external_service/aws_codecommit.md]: You could condense these steps to:

1. Go to **Site admin** > **Manage repositories** > **Add repositories**
1. Select **AWS CodeCommit**

File: doc/admin/external_service/bitbucket_cloud.md [hunk 3]
```diff
@@ -5,8 +5,8 @@ Site admins can sync Git repositories hosted on [Bitbucket Cloud](https://bitbuc
 To set this up, add Bitbucket Cloud as an external service to Sourcegraph:
 
 1. Go to **User menu > Site admin**.
-1. Open the **External services** page.
-1. Press **+ Add external service**.
+1. Open the **Repositories > Manage repositories** page.
+1. Press **+ Add repositories**.
 1. In the list, select **Bitbucket.org repositories**.
 1. Enter a **Display name** (using "Bitbucket Cloud" is OK).
```
Comment (ryan-blunden) [inline doc/admin/external_service/bitbucket_cloud.md]: I don't think you need this step as it's pretty obvious and pre-filled.

File: doc/admin/external_service/bitbucket_cloud.md [hunk 4]
```diff
@@ -5,8 +5,8 @@ Site admins can sync Git repositories hosted on [Bitbucket Cloud](https://bitbuc
...
 1. Configure the connection to Bitbucket Cloud in the JSON editor. Use Cmd/Ctrl+Space for completion, and [see configuration documentation below](#configuration).
```
Comment (ryan-blunden) [inline doc/admin/external_service/bitbucket_cloud.md]:
```suggestion
1. Configure the connection to Bitbucket Cloud using the action buttons above the text field, and additional fields can be added using **Cmd/Ctrl+Space** for auto-completion. See the [configuration documentation below](#configuration).
```

**Comment by unknwon** [comment]
Thanks @ryan-blunden for great feedback! Merging.

**Pull request merged**